### PR TITLE
Add Who said it best

### PR DESCRIPTION
**Pull request by pangyuehung**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,10 +8,10 @@ source of examples for all Elm developers.
 ## Submissions
 
 Simply submit a PR with your info added to the beginning of the array in
-`data/projects.json` and add a screenshot in `data/images`. Ensure that the
-screenshot is **1000px x 800px** and is **optimized** for the web. If your
-project is open source, please fill in your repository URL. Otherwise, just
-leave it as `null`.
+`public/data/projects.json` and add a screenshot in `public/data/images`.
```
Comment (dwayne) [inline README.md:11]: Thanks for noticing and fixing the incorrect directory references.

File: public/data/projects.json [hunk 2]
```diff
@@ -1,4 +1,11 @@
 [
+  {
+    "name": "Who Said It Best?",
+    "description": "Who Said It Best? is a simple quiz game to differentiate between a contemporary billionaire from the greatest tyrant of the last century."
```
Comment (dwayne) [inline public/data/projects.json:4]: You're missing a comma at the end.

File: public/data/projects.json [hunk 3]
```diff
@@ -1,4 +1,11 @@
...
+    "primaryUrl": "https://muskorhitler.com"
```
Comment (dwayne) [inline public/data/projects.json:5]: Same.

File: public/data/projects.json [hunk 4]
```diff
@@ -1,4 +1,11 @@
...
+    "previewImageUrl": "data/images/who-said-it-best.png",
+    "repositoryUrl": "https://gitlab.com/pangyuehung/who-said-it-best"
+  }
```
Comment (dwayne) [inline public/data/projects.json:8]: Same.

**Review by dwayne — CHANGES_REQUESTED** [review]

**Comment by dwayne** [comment]
@pangyuehung I reviewed your app and it looks like your [Learn more about actual socialism](https://muskorhitler.com/url) link doesn't work.

Thanks for your contribution. However, I don't think I will be including this app on the site. If you want to submit a separate PR for the README changes that will be fine.

**Comment by pangyuehung** [comment]
> link doesn't work

Fixed. No problem, thanks for the consideration

**Pull request closed without merging**